packages/airnode-node/src/handlers/process-http-signed-relayed-request.ts

```diff
@@ -0,0 +1,65 @@
+import find from 'lodash/find';
+import * as wallet from '../evm/wallet';
+import { AggregatedApiCall, Config, WorkerOptions, ApiCallSuccessResponse } from '../types';
+import * as logger from '../logger';
+import { go } from '../utils/promise-utils';
+import { spawnNewApiCall } from '../adapters/http/worker';
+import { WORKER_CALL_API_TIMEOUT } from '../constants';
+
+export async function processHttpSignedRelayedRequests(
+  config: Config,
+  endpointId: string,
+  // TODO: This should be typed as Record<string, string | undefined>
+  parameters: Record<string, string>
+): Promise<[Error, null] | [null, ApiCallSuccessResponse]> {
+  const trigger = find(config.triggers.httpSignedRelayed, ['endpointId', endpointId]);
+  if (!trigger) {
+    return [new Error(`Unable to find endpoint with ID:'${endpointId}'`), null];
+  }
+
+  const endpoints = find(config.ois, ['title', trigger.oisTitle])?.endpoints;
+  const endpoint = find(endpoints, ['name', trigger.endpointName]);
+  if (!endpoint) {
+    return [new Error(`No endpoint definition for endpoint ID '${endpointId}'`), null];
+  }
+
+  // Check that the required relayer parameters have been supplied
+  // TODO: There should be an TS interface for required params
+  if (!parameters._id) {
+    return [new Error(`You must specify "id" for the requestId/subscriptionId in the request parameters.`), null];
+  }
+  if (!parameters._relayer) {
+    return [new Error(`You must specify "relayer" address in the request parameters.`), null];
+  }
```

Comment:
Actually, this requires a bit more changes on Airnode side, because the parameters are expected to be a flat structure (Record<string, string>). Also, this should be probably handled consistently to other types or requests (regular, http) - so that we can have an unified structure, e.g.:
```json
{
    "reservedParams": {"_type":...},
    "apiParams": {"some-api-param":...}
    "airnodeParams": {"relayer": ...}
}
```

But I'd handle this in a separate issue, WDYT @amarthadan

Reply:
Tracked in https://api3dao.atlassian.net/browse/AN-571